### ISSUE TITLE
Introduce a ToggleButtons button_color style attribute

### ISSUE THREAD

**Issue by simaoh** [jupyter-widgets/ipywidgets#1719]
Can we change color and font of the buttons in `ToggleButtons`?
Should this be added to the ToggleButtonsStyle class in `ipywidgets/widgets/widget_selection.py`? (as in https://github.com/jupyter-widgets/ipywidgets/issues/1189) Something like this:

```python 
from .widget_core import CoreWidget
from .widget_style import Style
from .trait_types import InstanceDict, Color

class ToggleButtonsStyle(Style, CoreWidget):
     """Button style widget."""
     _model_name = Unicode('ToggleButtonsStyleModel').tag(sync=True)
     button_width = Unicode(help="The width of each button.").tag(sync=True)
     button_color = Color('#060482', allow_none=True, help="Color of the button").tag(sync=True)
     font_weight = Unicode(help="Button text font weight.").tag(sync=True)
```

**Comment by pbugnion** [comment]
Sounds like a good idea to me!

**Comment by Doodledun** [comment]
@pbugnion I am at PyconCa Sprint, I am very interested in this issue. I am going to try to work on this one.  Thanks for the tips!

**Comment by jasongrout** [comment]
FIxed by #1834. Thanks again!
